packages/integration_test/lib/_callback_io.dart

```diff
@@ -54,9 +59,43 @@ class IOCallbackManager implements CallbackManager {
     // comes up in the future. For example: `WebCallbackManager.cleanup`.
   }
 
+  // Whether the Flutter surface uses an Image.
+  bool _usesFlutterImage = false;
+
   @override
-  Future<void> takeScreenshot(String screenshot) {
-    throw UnimplementedError(
-        'Screenshots are not implemented on this platform');
+  Future<void> convertFlutterSurfaceToImage() async {
+    await integrationTestChannel.invokeMethod<void>(
+      'convertFlutterSurfaceToImage',
+      null,
+    );
+    _usesFlutterImage = true;
```

Comment:
What's the impact of calling this multiple times?

Should there be an assertion here saying `assert(!_usesFlutterImage)`?

Reply:
Done